snowexsql/tables/measurement_type.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import Column, String, Integer, ForeignKey, Date
+from sqlalchemy.orm import relationship, declared_attr
+
+from .base import Base
+
+
+class MeasurementType(Base):
+    """
+    Table to store the measurement types
+    """
+    __tablename__ = 'measurements'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String())
+
+
+class Measurement(object):
+    """
+    Base Class providing attributes required for a measurement of any kind
+    """
+
+    @declared_attr
+    def measurement_id(cls):
+        return Column(Integer, ForeignKey('public.measurements.id'),
+                      index=True)
+
+    @declared_attr
+    def measurement(cls):
+        return relationship('MeasurementType')
+
+    units = Column(String(50))
+    # Date of the measurement
+    date = Column(Date)
```

Comment:
I moved date to here

Reply:
As far as I remember, date should stay with each measurement. I will add context to the issue #134

Reply:
(Moved my discussion for the date column from the issue to here)

I see the intention of date now. However, I think we can't blanket add this column to the three tables that inherit the `measurement_type` relationship. It will be true for the Image and Point data. For the Layer data I would argue we should store it with the Pit (which I think we will rename to site ??, TBD)

Reply:
Yeah that makes sense. I will move date into the individual tables